### ISSUE TITLE
Provide screen reader support for the filters

### ISSUE THREAD

**Issue by ways2read** [daisy/ace-gui#27]
Screen reader experience when setting and changing filters is poor.

System: Windows 10 build 1803, with NVDA and Narrator.

To replicate:
Run an Ace report
Move to the Violations tab and select it
Move further down to Filter by button which should be collapsed. Expand it (with space) and with a screen reader you hear "Expanded".
Tab into the filter boxes and you will hear "Edit".
There are four filter boxes across the screen, these are impact, rule set, rule and location. This text is shown on the screen in each edit box. As you move between the boxes it is not obvious with a screen reader which one you are in. With NVDA it says edit, Narrator is silent.
Alt + down opens a list. With NVDA the list immediately disappears if you try to move down it. With Narrator it behaves better, but the lists are completely silent.
If you manage to add a filter it is added to the edit box, but this is not announced by the screen reader. It is not possible to determine which filters are set.

**Comment by marisademeglio** [comment]
Development note: the current filters are created with react-select (https://react-select.com/home)

**Comment by marisademeglio** [comment]
There is a chance the component we use will get updated with better a11y support, but it's hard to tell from the issues what the timeline is:
https://github.com/JedWatson/react-select/issues/2456

**Comment by nvdaes** [comment]
Seems edit boxes are not labelled by their corresponding headings. Also, the tooltips read, for example "press arrows to open a menu" (to select an option like moderate and so on, are not localized (always read in English).
While this is nt fixed, baybe useful some kind of documentation explaining that screen readers may prefer to use browser mode (in NVDA), virtual cursor in JAWS, etc.
